src/components/Main/Main.js

```diff
@@ -57,7 +58,9 @@ export default class Main extends Component {
     const { categoryData } = this.state
     let render
     if (!categoryData.length) {
-      render = <h1 className='loading'>Loading...</h1>
+      render =  <div className="App">
+      <Loading />
+    </div>
```

Comment:
What exactly is this div around the Loading component doing? Just kind of confusing to have it be called App when were rendering it inside Main, which is inside of App. Maybe consider renaming it unless there's a reason not to, other than that looks good though!